src/main/scala/org/ergoplatform/api/ApplicationApiRoute.scala

```diff
@@ -0,0 +1,55 @@
+package org.ergoplatform.api
```

Comment:
Why isn't it in the `org.ergoplatform.http.api` package?

Reply:
fixed in 151d5b4f2e5eea5e6e5e07260f7fa97c56ace28e